var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -496,3 +511,28 @@ def check_install(self):
 
         output = compile_c_and_execute(source_file, [include_flags], link_flags.split())
         compare_output_file(output, blessed_file)
+
+
+class CMakeBuilder(spack.build_systems.cmake.CMakeBuilder, SetupEnvironment):
+    def cmake_args(self):
+        cmake_defs = [self.define("TARGET", "GENERIC")]
```

Comment:
You may have added a few of these options since, but I still don't see handling of `ilp64`, `consistent_fpcsr`, `locking`, or `bignuma` (so I'd still be inclined not to make `CMake` the default on systems other than Windows).